src/ansys/sherlock/core/types/model_types.py

```diff
@@ -0,0 +1,65 @@
+# © 2024 ANSYS, Inc. All rights reserved.
+
+"""Module containing types for the Model Service."""
+
+try:
+    import SherlockModelService_pb2
+except ModuleNotFoundError:
+    from ansys.api.sherlock.v0 import SherlockModelService_pb2
+
+model_service = SherlockModelService_pb2
+
+
+class ExportFEAModelAnalysisType:
+    """Constants for type of analysis in the Export FEA Model request."""
+
+    __analysis_type = model_service.ExportFEAModelRequest.ExportAnalysis
+    NATURAL_FREQUENCY = __analysis_type.NaturalFreq
+    HARMONIC_VIBE = __analysis_type.HarmonicVibe
+    ICT = __analysis_type.ICTAnalysis
+    MECHANICAL_SHOCK = __analysis_type.MechanicalShock
+    RANDOM_VIBE = __analysis_type.RandomVibe
+
+
+class MinHoleDiameter:
```

Comment:
How about creating one class that stores a value and unit? E.g. `Measurement`.

If not, I recommend either renaming these constants to indicate they are only used for Export FEA Model (e.g. ExportFEAModelMaxEdgeLength) or not specify where these are to be used.

I prefer having a single class since all measurements in Sherlock have a value and units.